src/main/java/com/solvd/noteservice/web/WebConfig.java

```diff
@@ -13,7 +13,7 @@ public class WebConfig {
 
     @Bean
     @LoadBalanced
-    final RestTemplate restTemplate() {
+    RestTemplate restTemplate() {
```

Comment:
@vpetrovaa why this method is marked as `public`?

Reply:
@vpetrovaa I got the exception when it was final

Reply:
@vpetrovaa Yes, I know, but I mean why it's not `public` method?

Reply:
understand, https://github.com/vpetrovaa/note-service/pull/15/commits/8191b5a5c04e398dd17d031e6545b63b6eb90a32